src/_scss/pages/award/shared/awardAmountsSection/_dataTable.scss

```diff
@@ -47,18 +47,17 @@
         border: rem(2) solid #6E338E;
     }
 
-    // [DEV-5309] update colors for post cares act release
-    // .award-amounts__potential {
-    //     background-color: rgb(170, 198, 226);
-    // }
+    .award-amounts__potential {
+        background-color:#AAC6E2;
+    }
 
-    // .award-amounts__current {
-    //     background-color: rgb(85, 142, 198);
-    // }
+    .award-amounts__current {
+        background-color: rgb(85, 142, 198);
```

Comment:
Is there a reason for using `rgb` here instead of `#`?

Reply:
a6537ce